features/routing/haproxy-router.feature

```diff
@@ -174,12 +174,12 @@ Feature: Testing haproxy router
 
     Given I switch to the first user
     And I have a project
-    Given I obtain test data file "routing/list_for_caddy.json"
-    When I run the :create client command with:
-      | f | list_for_caddy.json |
+    Given I obtain test data file "routing/web-server-rc.yaml"
+    When I run oc create over "web-server-rc.yaml" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 2 |
```

Comment:
the old list_of_caddy.json uses `replicas: 2` and the new web-server-rc.yaml uses `replicas: 1`, but the `backend health check interval` cases need at least two backends, so `replacing paths` is required here.